src/ophys_etl/transforms/classification.py

```diff
@@ -0,0 +1,304 @@
+import h5py
+import boto3
+import json
+import math
+import joblib
+import numpy as np
+from urllib.parse import urlparse
+from scipy.sparse import coo_matrix
+from argschema import ArgSchema, ArgSchemaParser, fields
+from ophys_etl.schemas.fields import H5InputFile
+from croissant.features import FeatureExtractor
+from botocore.errorfactory import ClientError
+from marshmallow import post_load, ValidationError, Schema, validates
+import marshmallow.fields as mm_fields
+import warnings
+import os.path
+
+
+NOT_CELL_EXCLUSION_LABEL = -99999
+
+
+class RoiJsonSchema(Schema):
+    """ Schema of individual ROI records """
+    x = mm_fields.Int(required=True)
+    y = mm_fields.Int(required=True)
+    width = mm_fields.Int(required=True)
+    height = mm_fields.Int(required=True)
+    exclusion_labels = mm_fields.List(mm_fields.Int(), required=True)
+    mask_matrix = mm_fields.List(
+        mm_fields.List(mm_fields.Boolean()), required=True)
+    max_correction_up = mm_fields.Int(required=True)
+    max_correction_down = mm_fields.Int(required=True)
+    max_correction_left = mm_fields.Int(required=True)
+    max_correction_right = mm_fields.Int(required=True)
+    valid_roi = mm_fields.Boolean(required=True)
+    coo_roi = mm_fields.Field(required=False, load_only=True)
+
+    @post_load
+    def add_coo_data(self, data, **kwargs):
+        """Convert ROIs to coo format, which is used by the croissant
```

Comment:
It might help to include somewhere in here prominently the word `cropped` or `bounding box` to make it jump out what you're explaining in detail below.